hazelcast-client/src/test/java/com/hazelcast/client/cache/nearcache/ClientNearCacheInvalidationTest.java

```diff
@@ -0,0 +1,355 @@
+/*
```

Comment:
`ClientCacheClearTest` is also a functional test for `cache.clear`, if you don't mind I'd rather keep this test under the `nearcache` sub-package as it has to do exclusively with near cache functionality.